ios/Classes/TiNfcNfcAdapterProxy.h

```diff
@@ -7,13 +7,16 @@
 
 #if IS_IOS_11
 
+#import "NativeTagTechnologyProxy.h"
 #import "TiProxy.h"
 #import <CoreNFC/CoreNFC.h>
 
-@interface TiNfcNfcAdapterProxy : TiProxy <NFCNDEFReaderSessionDelegate> {
+@interface TiNfcNfcAdapterProxy : TiProxy <NFCNDEFReaderSessionDelegate, NFCTagReaderSessionDelegate> {
   NFCNDEFReaderSession *_nfcSession;
+  NFCTagReaderSession *_nfcTagReadersession;
   KrollCallback *_ndefDiscoveredCallback;
   KrollCallback *_nNdefInvalidated;
+  NativeTagTechnology *tagTech;
```

Comment:
Why we are storing tagTechnology object in property ?

Reply:
removed